### PR TITLE
Developers Portal codeowners: assign B9lab team as codeowners

### PR DESCRIPTION
**Pull request by barriebyron**
Now that we have our fabulous new Developers Portal, let's ensure that the PRs are able to be merged.

In past cycles, @toschdev and I relied only on each other to merge PRs, and this bottleneck was ineffective for getting fixes published quickly.

Let's assign the B9lab team as codeowners.

We can use pattern matching for relevant content reviewers. For now, PR reviewers can be selected for each PR if these auto codeowner assignments don't work. it's a start! 

See https://help.github.com/articles/about-codeowners/ for details. 

See IBC CODEOWNERS for a good example of pattern matching by content type

https://github.com/cosmos/ibc-go/blob/main/.github/CODEOWNERS

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# Docs 
+*.md @citmc @cmwaters @nooomski @coldice
```
Comment (coldice) [inline .github/CODEOWNERS:4]:
```suggestion
*.md @citmc @cmwaters @nooomski @xavierlepretre @coldice 
```
We would like to add Xavier here

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# Docs 
+*.md @citmc @cmwaters @nooomski @coldice 
+
+# Primary repo maintainers
+* @coldice @gioelebonifetto @8bitpal
```
Comment (coldice) [inline .github/CODEOWNERS:7]:
```suggestion
* @coldice @xavierlepretre @gioelebonifetto @8bitpal 
```
Also here, adding Xavier

Comment (nooomski) [reply .github/CODEOWNERS:7]: @okwme wdyt, do we need someone from IG/AiB on this list as well?

Comment (barriebyron) [reply .github/CODEOWNERS:7]: @toschdev and I struggled for so long as the only two approvers... 
hi @okwme !! 
CODEOWNERS assigns the reviewer automatically. You can choose to add reviewers at will to any PR.

Comment (nooomski) [reply .github/CODEOWNERS]: Yeah ok cool let's just go ahead with this and iterate when needed.

**Review by nooomski — APPROVED** [review]
👍

**Review by toschdev — APPROVED** [review]

**Pull request merged**